packages/office-ui-fabric-react/src/components/Rating/__snapshots__/Rating.test.tsx.snap

```diff
@@ -2,209 +2,251 @@
 
 exports[`Rating Renders Rating correctly 1`] = `
 <div
-  className="ms-Rating"
-  role="application"
+  aria-label=""
+  className="ms-Rating-star"
 >
   <div
+    aria-describedby={undefined}
     aria-labelledby={undefined}
-    className="ms-Rating-container"
-    role="radiogroup"
+    className="ms-FocusZone ms-Rating-focuszone"
+    data-focuszone-id="FocusZone2"
+    onFocus={[Function]}
+    onKeyDown={[Function]}
+    onMouseDownCapture={[Function]}
+    role="presentation"
+    tabIndex={-1}
   >
-    <div
-      className="ms-Rating-star is-selected undefined"
+    <button
+      className="ms-Rating-button ms-Rating--small undefined"
+      data-is-current={true}
+      disabled={false}
+      onFocus={[Function]}
+      role="presentation"
     >
-      <input
-        aria-labelledby="RatingLabel1-1"
-        checked={true}
-        className="ms-Rating-input"
-        disabled={undefined}
-        id="Rating0-1"
-        name="Rating0"
-        onBlur={[Function]}
-        onChange={[Function]}
-        onFocus={[Function]}
-        type="radio"
-        value={1}
-      />
-      <label
-        className="ms-Rating-label"
-        htmlFor="Rating0-1"
+      <span
+        className="ms-Rating-labelText"
+        id="RatingLabel1-1"
       >
-        <span
-          className="ms-Rating-labelText"
-          id="RatingLabel1-1"
-        >
-          1 Star
-        </span>
+        1 of 5 Star selected
```

Comment:
Here's a perfect example of why it's terrible to do what you're doing. It's incorrect grammar in English to say "1 of 5 Star selected". "1 of 5 stars selected" would be correct, no Title casing the subject... but it's going to be way different in Hebrew.

Reply:
Fixed